testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_delta_sync.py

```diff
@@ -128,6 +145,9 @@ def test_delta_sync_replication(params_from_base_test_setup, num_of_docs, replic
         assert delta_size < doc_writes_bytes, "did not replicate just delta"
 
     sg_docs = sg_client.get_all_docs(url=sg_admin_url, db=sg_db, include_docs=True)["rows"]
+    doc_ids = db.getDocIds(cbl_db)
+    cbl_db_docs = db.getDocuments(cbl_db, doc_ids)
+    print("all cbl docs are ----))))))-----", cbl_db_docs)
```

Comment:
is this print only for debug only during coding?

Reply:
done, removed the print statement. Added for debugging, forgot to remove